### ISSUE TITLE
Azure Environment Selector

### ISSUE THREAD

**Issue by jimseiwert** [microsoft/vscode-azurelogicapps#133]
It looks like this extension is only connecting to Azure Commercial and not respecting the azure environment selector of the azure plugin. Is there anyway we can set the environment to use Azure US Government Cloud?

**Comment by creativekinetix** [comment]
@joechung-msft where does this change need to be made? Is it an issue with the implementation of vscode-azure-account or vscode-azuretools? If you can provide some direction that would be helpful. I'm currently trying to track down why the Logic App extension isn't respecting the selected Azure Cloud (ie: Azure Government). All other VSCode Azure extensions seem to read this properly.

Thanks,

**Comment by creativekinetix** [comment]
@joechung-msft Fixed and confirmed to be connecting to Azure Commercial and Azure US Government. Will be submitting PR shortly.

**Comment by creativekinetix** [comment]
This issue is being caused by the version of `vscode-azureextensionui` being used by the the extension and can't be fully resolved.

I'm able to make changes to the code and connect to logic apps in Government Cloud but none of the other commands work (ie: Create Logic App) because they rely on the ui extension which is defaulting to Azure Commercial resource manager endpoint.

@joechung-msft, are there currently any plans to update the `vscode-azureextensionsui` module? I was taking a look over the weekend and appears there are significant breaking changes in v0.17.0+. I would be happy to help out with this effort.

**Comment by joechung-msft** [comment]
Will updating `vscode-azureextensionui` unblock progress on this?  In the meantime, I have forwarded this PR to my lead and PM for them to review.

**Comment by creativekinetix** [comment]
I will cleanup and resubmit the PR. I tried making the changes to fix the wizard errors caused by `vscode-azureextentsionui` so I want to remove those.

As for the update, the upgrade I believe would fix this issue completely, however as I mentioned, there are significant breaking changes which appear would require a good amount of rework/rewriting because a lot of the interfaces have been moved into abstract classes with slightly different implementations etc.

I started to make the necessary changes but wanted to make sure I wasn't duplicating work if there was already an effort being made. I will share the repo once I have a working base for the upgrade. Would surely welcome input on direction etc.

Thanks!

**Comment by joechung-msft** [comment]
You aren't duplicating work.  Your effort thus far is greatly appreciated!

**Comment by creativekinetix** [comment]
@joechung-msft Good news, updating to 0.17.1 was a pretty easy lift. I just submitted a new PR. Please disregard #150 and use PR #151 instead.

Let me know if you have questions or need anything additional from me.

**Comment by jamesseiwert** [comment]
Any update on this PR #151 ? It looks like this bug is still active
